Makefile

```diff
@@ -1,63 +1,139 @@
+#
+# Makefile for the Links Programming Language.
+#
+
+# This file contains instructions for building a bleeding-edge version
+# of Links in release or development mode with and without database
+# support.
+
+# LATEST STABLE RELEASE
```

Comment:
This is now slightly duplicated from the INSTALL file. I feel like we should just refer readers there for how to install the stable version

Reply:
Done.